packages/prerelease-registry/functions/routes/index.ts

```diff
@@ -0,0 +1,10 @@
+import index from "./index.html";
```

Comment:
Ugh sorry. I forgot to mention that to keep TypeScript happy we have to add an extra file:

**html.d.ts**
```ts
declare module '*.html' {
  const content: string;
  export default content;
}
```

Reply:
Fixed!